Rectangle/Snapping/SnappingManager.swift

```diff
@@ -306,14 +306,14 @@ class SnappingManager {
                 let thirdWidth = floor(frame.width / 3)
                 if loc.x >= frame.minX + cornerSize && loc.x <= frame.minX + thirdWidth {
                     if let priorAction = priorSnapArea?.action {
-                        if priorAction == .topLeft || priorAction == .topLeftSixth {
+                        if priorAction == .topLeft || priorAction == .topLeftSixth || priorAction == .topCenterSixth {
                             return SnapArea(screen: screen, action: .topLeftSixth)
                         }
                     }
                 }
-                if loc.x >= frame.minX + thirdWidth && loc.x <= frame.maxX - cornerSize {
+                if loc.x >= frame.maxX - thirdWidth && loc.x <= frame.maxX - cornerSize {
```

Comment:
Not sure why the top center was being treated as part of the top right sixth snap zone, but it was interfering with the new top center sixth snap zone.

Reply:
I may have just got confused 😁